gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -778,6 +779,13 @@ public GradeSaveResponse saveGrade(final Long assignmentId, final String student
 			log.error("An error occurred saving the grade. {}: {}", e.getClass(), e.getMessage());
 			rval = GradeSaveResponse.ERROR;
 		}
+
+		try {
+			EventHelper.postUpdateGradeEvent(gradebook, assignmentId, studentUuid, newGrade, rval, getUserRole(getCurrentSiteId()));
+		} catch (GbAccessDeniedException e) {
+			// oh well
```

Comment:
this will cause a sonar error, can you please log.debug this if its not an issue that an event doesn't happen. is anything relying on events and what would cause a failure?